src/app/contact/page.tsx

```diff
@@ -0,0 +1,11 @@
+import { FC } from 'react'
+
+interface ContactPageProps {
+
+}
+
+const ContactPage: FC<ContactPageProps> = ({}) => {
+  return <div>Contact page</div>
+}
+
+export default ContactPage
```

Comment:
Not a blocker for now, but just noticed this - make sure you get the prettier extension installed in your vscode, and probably change your settings to Format on Save. I'll get a prettierrc file in here later.

tl;dr - it's good to use an autoformatter, and if one person on a project doesn't then you end up in this situation where the next person in this file is going to create a bunch of whitespace changes when their formatter adds semicolons to this whole file.